peps/pep-0731.rst

```diff
@@ -5,10 +5,12 @@ Author: Guido van Rossum <guido@python.org>,
         Victor Stinner <vstinner@python.org>,
         Steve Dower <steve.dower@python.org>,
         Irit Katriel <irit@python.org>
+Discussions-To: https://discuss.python.org/t/pep-731-c-api-working-group-charter/36117
 Status: Draft
 Type: Process
-Created: 11-Oct-2023
+Created: 13-Oct-2023
```

Comment:
```suggestion
Created: 11-Oct-2023
```
It was still created on the 11th, not that it matters much.

Reply:
Sure, I wasn't sure what the convention is for Created.